napari_organoid_counter/_widget.py

```diff
@@ -112,9 +120,98 @@ def __init__(self,
         self.viewer.layers.events.inserted.connect(self._added_layer)
         self.viewer.layers.events.removed.connect(self._removed_layer)
         self.viewer.layers.selection.events.changed.connect(self._sel_layer_changed)
+    
         # setup flags used for changing slider and text of min diameter and confidence threshold
         self.diameter_slider_changed = False 
-        self.confidence_slider_changed = False 
+        self.confidence_slider_changed = False
+
+        # Key binding to return the ID of the bounding boxes selected from the GUI
+        @self.viewer.bind_key('s')
+        def selected_boxes(viewer: napari.Viewer):
+            if self.cur_shapes_layer is not None:  # Check if there is a valid shapes layer selected
+                selected_shapes = self.cur_shapes_layer.selected_data
+                print(f"Selected shapes indices: {selected_shapes}")
+
+        # Dictionary where key are shape indices and values are RGBA colors
+        self.original_colors = {}
+
+        # Key binding to change the edge_color of the bounding boxes to green
+        @self.viewer.bind_key('g')
+        def change_edge_color_to_green(viewer: napari.Viewer):
+            if self.single_annotation_mode:  # Check if single-annotation mode is active
+                show_info("Cannot change edge color. Change to multi-annotation mode to enable this feature.")
+                return
+            
+            if self.cur_shapes_layer is not None:  # Ensure shapes layer exists
+                selected_shapes = self.cur_shapes_layer.selected_data # Retrieves indices of shapes currently selected, returns a set 
+                if len(selected_shapes) > 0:
+                    # Set the edge color of selected shapes to green
+                    new_edge_color = [85/255, 1., 0, 1.]  # RGBA for green
```

Comment:
We could save the colors in settings.py as global variables, and always refer to that variable each time we refer to it (maybe call them color_claa_1 and color_class_2). This is more robust and makes it easier to apply changes.